### PR TITLE
addition of Terms for user

### PR DESCRIPTION
**Pull request by jh-bate**
add terms to be stored for a user account

### CODE REVIEW

File: models/user_test.go [hunk 1]
```diff
@@ -49,6 +49,17 @@ func Test_Emails(t *testing.T) {
...
 	}
 }
+
+func Test_Terms(t *testing.T) {
+
+	terms := "2015/03/15"
+	user := UserFromDetails(&UserDetail{Terms: terms})
+
+	if user.Terms != terms {
+		t.Fatalf("there terms date should have been set")
```
Comment (jebeck) [inline models/user_test.go]: "there" should be "the" here? just a bit confusing

**Comment by jh-bate** [comment]
@darinkrauss will run this past you before I merge

File: user/user.go [hunk 2]
```diff
@@ -12,6 +12,7 @@ type User struct {
 	Id       string                 `json:"userid" bson:"userid,omitempty"` // map userid to id
 	Name     string                 `json:"username" bson:"username"`
 	Emails   []string               `json:"emails" bson:"emails"`
+	Terms    string                 `json:"terms" bson:"terms"`
```
Comment (darinkrauss) [inline user/user.go]: It isn't clear to me what `Terms` means.  After I read the rest of the code I thought `TermsAccepted` might be clearer.

File: user/api.go [hunk 3]
```diff
@@ -297,6 +297,11 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
 				}
 			}
 
+			//Updated Terms
+			if updatesToApply.Updates.Terms != "" {
```
Comment (darinkrauss) [inline user/api.go]: Will `updatesToApply.Updates.Terms` ever be nil?

**Comment by darinkrauss** [comment]
A couple of minor questions, but otherwise LGTM.  :ship:

File: user/api.go (hunk 3 above)
Comment (jh-bate) [inline user/api.go]: a string can't be Nil only empty

File: user/user.go (hunk 2 above)
Comment (jh-bate) [inline user/user.go]: agreed - have updated

**Pull request merged**